CMakeLists.txt

```diff
@@ -58,6 +58,10 @@ option(SWIFT_INCLUDE_DOCS
     "Create targets for building docs."
     TRUE)
 
+option(SWIFT_ANALYZE_CODE_COVERAGE
```

Comment:
I think `OFF` would be the best of the various CMake names for false, but we don't use that anywhere else, so I'll let you decide. I just want to make sure it's clear that `TRUE` isn't valid here!